modules/osint/username_search.py

```diff
@@ -58,15 +59,18 @@ def check(self, url, site, data):
 			OUTPUT['links'][site] = url
 
 def module_api(self):
-	query = self.options['query']
-	filepath = os.path.join(os.getcwd(), 'data', 'username_checker.json')
-	with open(filepath) as file:
-		data = json.loads(file.read())
-	thread(self, data, query,self.options['thread'])
-	output = OUTPUT
+        query = self.options['query']
```

Comment:
Replace 4 spaces with \t.

Reply:
yes my bad vim screwed up my tabs
closing this pr and opening another one